routes/qrcode.js

```diff
@@ -422,11 +425,19 @@ router.get(
         },
         order: [['pk', 'DESC']],
       })
-        .then(rows => {
+        .then(async rows => {
+          //create array of URL data
+          const qrcodes = [];
+          for (var i = 0; i < rows.length; i++) {
+            var res2 = await QRCode.toDataURL(rows[i].qrcode_url);
```

Comment:
@DanielmAaron  perhaps rename res2 to something more meaningful like qrcode_image